wurst/systems/trade/TradeShipCoordinator.wurst

```diff
@@ -0,0 +1,39 @@
+package TradeShipCoordinator
+import TradeShipList
+import TradeShipSupervisor
+import TradeShipRoutePair
+import LinkedList
+
+public class TradeShipCoordinator
+    var nextShip = TradeShipList.getFirst()
+    let routes = new LinkedList<TradeShipRoutePair>()
+    var ships = new LinkedList<TradeShipSupervisor>()
+
+    function swap()
+        routes.forEach(route -> route.swap())
+
+    function spawn()
+        ships.forEach(ship -> destroy ship)
+        destroy ships
```

Comment:
Does the destructor of a list not automatically destruct each member?

Reply:
@jonathanmclaus no, and thankfully for that. It would be very inconvenient.